lib/dartdoc.dart

```diff
@@ -128,7 +129,7 @@ class DartDoc {
   List<LibraryElement> _parseLibraries(List<String> files) {
     Set<LibraryElement> libraries = new Set();
     DartSdk sdk = new DirectoryBasedDartSdk(new JavaFile(sdkDir.path));
```

Comment:
You might consider moving SDK creation into the else block in case there is embedder information and it isn't needed?